README.md

```diff
@@ -101,14 +101,188 @@ These can be used in the `FromBauble` macro, and can be automatically implemente
 ```rust
 #[derive(FromBauble)]
 struct Button {
-  #[bauble(attribute)]
-  width: u32,
-  #[bauble(attribute)]
-  height: u32,
-  on_press: Action,
+    #[bauble(attribute)]
+    width: u32,
+    #[bauble(attribute)]
+    height: u32,
+    on_press: Action,
 }
 ```
 
+## `FromBauble` derive macro attributes
+
+### `attribute`
+
+See above.
+
+### `default`
+
+```rust
+// Rust
+#[derive(FromBauble)]
+struct Background {
+    content: Box<Widget>,
+    #[bauble(default = Color::Black)]
+    color: Color,
+    #[bauble(default)]
+    margin: Margin,
+}
+
+// bauble
+background = Background {
+    content: Widget::Text(Text("Hello World")),
+}
+```
+
+On a field, allows it to be omitted and replaced with a default expression. If no expression is
+provided, that field's `Default` implementation will be used.
+
+### `from_default`
+
+```rust
+#[derive(FromBauble)]
+struct Enemy {
+    name: String,
+    hp: u32,
+    damage: u32,
+    #[bauble(from_default = Uuid::new())]
+    id: Uuid,
+}
+```
+
+On a field, it will not be parsed and instead it will use the provided expression as the default
+value. If no expression is provided, that field's `Default` implementation will be used. The field's
+type does not need to implement `FromBauble`.
```

Comment:
I think this would be more clear if "as the default value" was removed. Since it being the default implies that there are scenarios where it isn't used.